samples/drivers/adc/sample.yaml

```diff
@@ -26,6 +26,8 @@ tests:
       - gd32l233r_eval
       - lpcxpresso55s36
       - mr_canhubk3
+      - longan_nano
+      - longan_nano_lite
```

Comment:
Does longan_nano_lite is different board than langan_nano? If so we need lite dts overlay too.

Reply:
I will update it. Thanks!